ccd-definition/FixedLists/CareSupervision/ManageOrdersType.json

```diff
@@ -89,5 +89,12 @@
     "ListElementCode": "C35A_SUPERVISION_ORDER",
     "ListElement": "Supervision order (C35A)",
     "DisplayOrder": 14
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ManageOrdersType",
+    "ListElementCode": "C36_VARIATION_OR_EXTENSION_OF_SUPERVISION_ORDERS",
+    "ListElement": "Variation or extension of supervision orders (C36)",
+    "DisplayOrder": 15
```

Comment:
same thing here, C34B is using 15, if this is getting pushed out before DFPL-43 then leave this here otherwise might be good to change the display order to accommodate that